services/src/messaging/src/main/java/org/openkilda/messaging/command/reroute/RerouteAllFlowsForIsl.java

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.command.reroute;
+
+import org.openkilda.messaging.info.event.PathNode;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class RerouteAllFlowsForIsl extends RerouteFlows {
```

Comment:
do we really need new RerouteFlows subtype here? why can't we utilize reroute affected flows instead?

Reply:
We should not reroute affected flows in this case. We have to reroute all flows which go through a particular ISL.